src/diffusers/pipelines/latent_consistency_models/pipeline_latent_consistency_img2img.py

```diff
@@ -179,6 +180,7 @@ def __init__(
         scheduler: LCMScheduler,
         safety_checker: StableDiffusionSafetyChecker,
         feature_extractor: CLIPImageProcessor,
+        image_encoder: Optional[CLIPVisionModelWithProjection] = None,
```

Comment:
@sayakpaul The line you're looking for is present here. I think you're looking at the wrong commit (that was merged to diffusers for StableDiffusionControlNetImg2ImgPipeline in #5901)